octobot_trading/constants.py

```diff
@@ -99,6 +99,9 @@
 BALANCE_PROFITABILITY_CHANNEL = "BalanceProfitability"
 POSITIONS_CHANNEL = "Positions"
 
+# 946742400 is 01/01/2020, if trade time is lower, there is an issue.
+MINIMUM_VAL_TRADE_TIME = 946742400
```

Comment:
If we load user exchange history, couldn't there be a trade in 2019 ?

Reply:
oh ! comment is wrong, its 01/01/2000

Reply:
Oh ok :smile: that's ok then :wink: